product/models.py

```diff
@@ -0,0 +1,91 @@
+from django.db import models
+from order.models import *
+from board.models import *
+from user.models import *
+
+class Category(models.Model):
+    name = models.CharField(max_length=45)
+    class Meta:
+        db_table = "categories"
+
+class Subcategory(models.Model):
+    name     = models.CharField(max_length=45)
+    category = models.OneToOneField(
+                'Category',
+                on_delete   = models.CASCADE,
+                )
+    class Meta:
+        db_table = "subcategories"
+
+class Character(models.Model):
+    name        = models.CharField(max_length=45)
+    image_url   = models.CharField(max_length=2000, default='')
+    class Meta:
+        db_table = "characters"
+
+class Product(models.Model):
+    name            = models.CharField(max_length=45, null=True)
+    price           = models.DecimalField(max_digits= 6, decimal_places=2)
+    star_rating     = models.DecimalField(max_digits= 1, decimal_places=1)
+    description     = models.TextField()
+    category        = models.OneToOneField(
+                        'Category',
+                        on_delete   = models.CASCADE,
+                        )
+    subcategory     = models.OneToOneField(
+                        'Subcategory',
+                        on_delete   = models.CASCADE,
+                        )
+    character       = models.OneToOneField(
+                        'Character',
+                        on_delete   = models.CASCADE,
+                        )
+    discount        = models.OneToOneField(
+                        'Discount',
+                        on_delete   = models.CASCADE,
+                        )
+    image           = models.CharField(max_length = 1000)
+    created_at      = models.DateTimeField(auto_now = True)
+    class Meta:
+        db_table = "products"
+
+class ProductImage(models.Model):
+    product     = models.ForeignKey(
+                    'Product',
+                    on_delete = models.CASCADE
+                    )
+    image_url   = models.CharField(max_length=2000)
+    class Meta:
+        db_table = "productimages"
+
+class Discount(models.Model):
+    discount_rate   = models.FloatField()
+    discount_type   = models.CharField(max_length=45)
+    discount_due    = models.DateTimeField(auto_now=True)
+    class Meta:
+        db_table = "discounts"
+
+class Review(models.Model):
+    product     = models.ForeignKey(
+                    'Product',
+                    on_delete = models.CASCADE
+                    )
+    member      = models.ForeignKey(
+                    'user.Member',
+                    on_delete = models.CASCADE
+                    )
+    content     = models.CharField(max_length=500)
+    created_at  = models.DateTimeField(
+                    auto_now = True,
+                    )
+    star_rating = models.DecimalField(max_digits= 1, decimal_places=1)
```

Comment:
```suggestion
    star_rating = models.DecimalField(max_digits= 1, decimal_places=1)
```
점수를 1.0 이런식으로 담으려면 max_digits를 3으로 해야할것 같은데 확인 부탁드립니다.